packages/vats/src/core/client-behaviors.js

```diff
@@ -112,8 +115,13 @@ export const startClient = async ({
     await E(httpVat).setCommandDevice(cmdDevice, roles);
     D(cmdDevice).registerInboundHandler(httpVat);
   }
+
+  const networkZone = makeDurableZone(
+    zone.detached().mapStore('networkZoneBaggage'),
+  );
+
   const addLocalPresences = async () => {
-    await registerNetworkProtocols(vats, undefined);
+    await registerNetworkProtocols(networkZone, vats, undefined);
```

Comment:
Re **Upgrade Considerations**: this change is in the boostrap vat, which we have no plans to upgrade. There's a goal, that I support, to avoid merging changes that we don't have plans to deploy.

This change looks like it was forced by a backwards-incompatible change to `registerNetworkProtocols`.

Did you consider adding the zone arg at the end instead of the beginning of the arg list, and making it optional, with a default of a heap zone?

Reply:
This is a great callout and find. I'll aim to make `registerNetworkProtocols` backwards compatibility